### DISCUSSION TITLE
[Feature] Add "relatable Images/Videos" to the item that you're currently seeing

### DISCUSSION THREAD

**Discussion by chriexpe** [immich-app/immich#3169] [Feature Request]
### The feature

This feature is similar to when you search a image on the web, you open it and below there is "relatable content".
On Immich it might be interesting to, when you have a photo/video open, scroll down the page (exactly like "stories") where it lists other photo/videos that have similar tag embeddings, this would fit great with this other feature request of "image interrogator" https://github.com/immich-app/immich/discussions/2519 that generates way more tags than the currently used "Clips".

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2447
